### PR TITLE
modified telemetry tests so that they do not include start event

### LINKED ISSUE #1139: broken CI due to telemetry changes
the CI is broken because we recently introduced a change to the [core](https://github.com/ploomber/core) library to perform fewer calls. should be an easy fix

```
 FAILED tests/telemetry/test_telemetry.py::test_install_lock_uses_telemetry[0-0-0-0-0-0]
FAILED tests/telemetry/test_telemetry.py::test_install_lock_uses_telemetry[0-0-0-1-0-0]
FAILED tests/telemetry/test_telemetry.py::test_install_lock_uses_telemetry[0-0-0-0-0-1]
FAILED tests/telemetry/test_telemetry.py::test_install_lock_uses_telemetry[0-0-0-1-0-1]
FAILED tests/telemetry/test_telemetry.py::test_install_lock_uses_telemetry[1-0-0-0-0-0]
FAILED tests/telemetry/test_telemetry.py::test_install_lock_uses_telemetry[1-0-0-1-0-0]
FAILED tests/telemetry/test_telemetry.py::test_install_lock_uses_telemetry[1-0-0-0-0-1]
FAILED tests/telemetry/test_telemetry.py::test_install_lock_uses_telemetry[1-0-0-1-0-1]
FAILED tests/telemetry/test_telemetry.py::test_install_uses_telemetry - Asser...
FAILED tests/telemetry/test_telemetry.py::test_build_uses_telemetry[expected0]
FAILED tests/telemetry/test_telemetry.py::test_task_command[args0] - Assertio...
FAILED tests/telemetry/test_telemetry.py::test_task_command[args1] - Assertio...
FAILED tests/telemetry/test_telemetry.py::test_task_command[args2] - Assertio...
FAILED tests/telemetry/test_telemetry.py::test_task_command[args3] - Assertio...
FAILED tests/telemetry/test_telemetry.py::test_task_command[args4] - Assertio...
FAILED tests/telemetry/test_telemetry.py::test_task_command[args5] - Assertio...
FAILED tests/telemetry/test_telemetry.py::test_task_command[args6] - Assertio...
FAILED tests/telemetry/test_telemetry.py::test_task_command[args7] - Assertio...
FAILED tests/telemetry/test_telemetry.py::test_status_command - AssertionErro...
FAILED tests/telemetry/test_telemetry.py::test_interact_uses_telemetry - Asse...
FAILED tests/telemetry/test_telemetry.py::test_parses_dag - IndexError: list ...
FAILED tests/telemetry/test_telemetry.py::test_parses_dag_on_exception - Inde...
= 22 failed, 2768 passed, 11 skipped, 5 xfailed, 1 xpassed, 291 warnings in 372.51s (0:06:12) =
```

### PR DESCRIPTION
**Pull request by lbellomo**
## Describe your changes

This fixes the telemetry tests to take into account the start event that was removed [this pr on core](https://github.com/ploomber/core/pull/77).

I couldn't get the test `test_install_uses_telemetry` to work local, but it works in the gh-actions. 

## Issue number

Closes #1139

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [x] Added [tests](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#testing) (when necessary).
- [x] Added [docstring](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#documenting-changes-and-new-features) documentation and update the [changelog](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#changelog) (when needed)

----
:books: Documentation preview :books:: https://ploomber--1141.org.readthedocs.build/en/1141/

### CODE REVIEW

File: tests/telemetry/test_telemetry.py [hunk 1]
```diff
@@ -313,7 +307,8 @@ def my_function(payload):
 
     my_function()
 
-    call2_kwargs = mock_posthog_capture.call_args_list[1][1]
+    print(mock_posthog_capture.call_args_list)
```
Comment (bryannho) [inline tests/telemetry/test_telemetry.py:310]: looks like you left a debug statement here. otherwise, lgtm!

**Review by bryannho — CHANGES_REQUESTED** [review]

File: tests/telemetry/test_telemetry.py (hunk 1 above)
Comment (lbellomo) [reply tests/telemetry/test_telemetry.py]: whops! yes, you are right!

**Comment by lbellomo** [comment]
Now the [windows actions is failing](https://github.com/ploomber/ploomber/actions/runs/6872256897/job/18690432227?pr=1141). Something completely different related to the installation in windows.

**Comment by bryannho** [comment]
> Now the [windows actions is failing](https://github.com/ploomber/ploomber/actions/runs/6872256897/job/18690432227?pr=1141). Something completely different related to the installation in windows.

Looks like it could be an intermittent error. Could you try to push an empty commit to retry the CI?

**Comment by lbellomo** [comment]
Sure.
It seems that the [action for mac](https://github.com/lbellomo/ploomber/actions/runs/6872256446/job/18690431327) also failed (but only for 3.9, for 3.10 it did work). And it seems that it is a different error.

**Comment by lbellomo** [comment]
Now [mac action](https://github.com/ploomber/ploomber/actions/runs/6872789273/job/18691808522) is failing with the same error that the previous one from [windows action](https://github.com/ploomber/ploomber/actions/runs/6872256897):

```
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    playwright from https://files.pythonhosted.org/packages/60/0e/9385adaf2bff5920e5967d1cc9a2eace6978a005792d163e79439c5205ae/playwright-1.31.1-py3-none-macosx_10_13_x86_64.whl (from nbconvert[webpdf]; extra == "dev"):
        Expected sha256 225ff39ff29219caab242a562e6a5efa486d4659ac10d16421ad5904c7f23b25
             Got        5fb1b405cb98723c162fb1fd44d7b42474a13fc4479221e3fa7b408e9b462d58
```

**Pull request merged**